### ISSUE TITLE
Update ANTLR dependency

### ISSUE THREAD

**Issue by emlyn** [venantius/glow#9]
Any chance you could update to clj-antlr 0.2.3? I'm using Spark 2.0 and it depends on ANTLR 4.5.3, which is causing conflicts.

**Comment by venantius** [comment]
Could you take a look at https://github.com/venantius/glow/pull/10 and confirm that it works for you? Note that you'll need to be on JDK 8 if you're not already.

**Comment by emlyn** [comment]
Thanks, I'm already on JDK 8. I'm not sure what's going on as I can't reproduce this anymore... instead I get the following error whenever I try to load the repl (even if I set `:ultra {:repl false, :stacktraces  false, :tests false}`):
```
ClassCastException clojure.lang.PersistentVector cannot be cast to java.lang.String
	clojure.core/read-string (core.clj:3677)
	clojure.core/read-string (core.clj:3677)
	reply.eval-modes.nrepl/completion-eval (nrepl.clj:178)
	reply.eval-modes.nrepl/completion-eval (nrepl.clj:169)
	clojure.core/partial/fn--4761 (core.clj:2522)
	reply.eval-modes.nrepl/main (nrepl.clj:224)
	reply.eval-modes.nrepl/main (nrepl.clj:199)
	reply.main/launch-nrepl/fn--11867 (main.clj:79)
	clojure.core/with-redefs-fn (core.clj:7216)
	clojure.core/with-redefs-fn (core.clj:7200)
	reply.main/launch-nrepl (main.clj:78)
	reply.main/launch-nrepl (main.clj:72)
```

**Comment by venantius** [comment]
I'm sorry, can you clarify? That doesn't seem like an issue that's related to Glow.

**Comment by emlyn** [comment]
Oh yes, you're right, I'm not using Glow directly, but through Ultra. It looks like that error is to do with Ultra, not Glow (it goes away when I remove Ultra). But it's preventing me from verifying the fix in #10.

**Comment by venantius** [comment]
What version of Ultra? Can you paste your `~/.lein/profiles.clj` and your `project.clj`?

**Comment by venantius** [comment]
I've released version 0.1.4 onto Clojars. Please take a look and confirm that it resolves the issue (this may necessitate disabling Ultra).

**Comment by emlyn** [comment]
Sorry, was a bit busy with other things and didn't get around to testing this until now. It all works nicely again, thanks!
